src/SME.SR.Data/Repositories/Sgp/FrequenciaAlunoRepository.cs

```diff
@@ -170,7 +170,7 @@ public async Task<IEnumerable<FrequenciaAluno>> ObterFrequenciaGlobalAlunos(stri
                             where fa.codigo_aluno = ANY(@codigosAluno) 
                               and t.ano_letivo = @anoLetivo
                               and t.modalidade_codigo = @modalidade 
-                              and fa.tipo = 1
+                              and fa.tipo = 2
                               and t.tipo_turma in(1,2,7) 
```

Comment:
Ficaria melhor para o enum (TipoFrequenciaAluno) para melhor entendimento do bug, 
exemplo : and fa.tipo = TipoFrequenciaAluno.Geral